src/data/library.json

```diff
@@ -10,7 +10,7 @@
       "slug": "the-shipping-news",
       "title": "The Shipping News",
       "author": "Annie Proulx",
-      "blurb": "The Shipping News is a novel by American author E. Annie Proulx and published by Charles Scribner's Sons in 1993. It won the <a href=\"https://www.pulitzer.org/winners/e-annie-proulx\">Pulitzer Prize</a>, the <a href=\"https://www.nationalbook.org/books/shipping-news/\">U.S. National Book Award</a>, as well as other awards. It was adapted as a film of the same name which was released in 2001."
+      "blurb": "The Shipping News is a novel by American author E. Annie Proulx and published by Charles Scribner's Sons in 1993. It won the Pulitzer Prize, the U.S. National Book Award, as well as other awards. It was adapted as a film of the same name which was released in 2001."
```

Comment:
Since we are pulling in the content from Google Docs as `text` we lose the HTML. You could perhaps bypass this by putting the html as plain text in the document, but I cannot modify the document so I did not attempt that.